OgreMain/src/OgreResourceGroupManager.cpp

```diff
@@ -109,6 +109,7 @@ namespace Ogre
         ScopedCLocale( bool changeLocaleTemporarily ) :
             mChangeLocaleTemporarily( changeLocaleTemporarily )
         {
+            memset( mSavedLocale, 0, sizeof( char ) * 64 );
```

Comment:
C++ standard guarantees that sizeof(char) is always equal 1. OTOH, it would be good to replace 64 by sizeof(mSavedLocale)